assets/js/googlesitekit/modules/datastore/sharing-settings.js

```diff
@@ -624,6 +687,44 @@ const baseSelectors = {
 	isDoingSubmitSharingChanges( state ) {
 		return !! state.isDoingSubmitSharingChanges;
 	},
+
+	/**
+	 * Gets the default sharing settings for shared ownership modules.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(Object|undefined)} Sharing Settings object. Returns undefined if it is not loaded yet.
+	 */
+	getDefaultSharedOwnershipModuleSettings( state ) {
+		const { defaultSharedOwnershipModuleSettings } = state;
+		return defaultSharedOwnershipModuleSettings;
+	},
+
+	/**
+	 * Indicates whether the sharing settings have updated from default.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {boolean} True if the sharing settings have updated, false otherwise.
+	 */
+	haveSharingSettingsUpdated: createRegistrySelector(
+		( select ) => ( state ) => {
+			const { savedSharingSettings } = state;
+
+			if ( ! savedSharingSettings || isEmpty( savedSharingSettings ) ) {
```

Comment:
Hmm, this could be reduced to `if ( isEmpty( savedSharingSettings ) )`, as `isEmpty` returns true for both `undefined` and `false`...

Reply:
Updated, thank you @techanvil!